index.bs

```diff
@@ -553,7 +553,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. For each |authenticator| currently available on this platform, if
     <code>|options|.{{MakeCredentialOptions/attachment}}</code> is [=present|not present=] or its value matches
-    |authenticator|'s attachment modality, [=set/append=] |authenticator| to |currentlyAvailableAuthenticators|.
+    |authenticator|'s attachment modality, [=set/append=] |authenticator| to |currentlyAvailableAuthenticators|. If 
```

Comment:
What's an "attachment modality"?  I tried to find another reference to "modality" in the spec, but failed.  Is it here somewhere and I just don't see it?